client/post-editor/editor-ground-control/index.jsx

```diff
@@ -299,28 +292,12 @@ export class EditorGroundControl extends PureComponent {
 						</span>
 					</div>
 				) }
-				{ ( isSaveAvailable || shouldShowStatusLabel ) && (
-					<div className="editor-ground-control__status">
-						{ isSaveAvailable && (
-							<button
-								className="editor-ground-control__save button is-link"
-								onClick={ this.onSaveButtonClick }
-								tabIndex={ 3 }
```

Comment:
The `tabIndex` seems to be lost from left to right here. What do you think about adding it as a prop to `SaveStatus` so we can keep all of these declarations in one place (i.e. `editor-ground-control/index.jsx`)?

Reply:
Ahh yes, good spot :) <3

Reply:
I've given this some thought and I'm not sure how this would best look.
Do you think it'd be fair to do something along these lines?:

```js

<SaveStatus tabIndex=7 />

// in SaveStatus

render() {
  return (
    <div className="wrapping-div">
      <ComponentWeWantToTabTo tabIndex={ this.props.tabIndex }
    </div>
```

I've found this pattern in use [this file](https://github.com/Automattic/wp-calypso/blob/master/client/components/section-nav/item.jsx) so it seems fair - just want to be a bit more confident in it before implementing :)

Reply:
Actually, I just implemented it :P